pkgs/development/python-modules/pygmt/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, gmt
+, numpy
+, netcdf4
+, pandas
+, packaging
+, xarray
+}:
+
+buildPythonPackage rec {
+  pname = "pygmt";
+  version = "0.2.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "GenericMappingTools";
+    repo = "pygmt";
+    rev = "v${version}";
+    sha256 = "1yx1n6mxfmwg69ls5560nm6d3jxyghv27981iplz7m7990bbp468";
+  };
+
+  postPatch = ''
+    substituteInPlace pygmt/clib/loading.py \
+      --replace "env.get(\"GMT_LIBRARY_PATH\", \"\")" "env.get(\"GMT_LIBRARY_PATH\", \"${gmt}/lib\")"
+  '';
+
+  propagatedBuildInputs = [ numpy netcdf4 pandas packaging xarray ];
+
+  doCheck = false; # requires network access
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Thanks, added.